### PR TITLE
Green departure time if realtime

### PR DESCRIPTION
**Pull request by csolem**
When real time is available for the itinerary, make the departure time green. This is a temporary solution until the full design is implemented.(with two departure times).

### CODE REVIEW

File: app/component/summary/summary-row.scss [hunk 1]
```diff
@@ -52,6 +52,10 @@
     margin-top: 4px;
   }
 
+  & .realtime-available {
+    color: #00985f;
```
Comment (hannesj) [inline app/component/summary/summary-row.scss]: This should probably come from the sass variable configured in the theme.

File: app/component/summary/summary-row.cjsx [hunk 2]
```diff
@@ -48,7 +51,7 @@ class SummaryRow extends React.Component
           {geoUtils.displayDistance(data.walkDistance)}
         </div>
       </div>
-      <div className="itinerary-start-time">
+      <div className={cx "itinerary-start-time", if realTimeAvailable then "realtime-available"}>
```
Comment (hannesj) [inline app/component/summary/summary-row.cjsx]: You can just use `{cx "itinerary-start-time", "realtime-available": realTimeAvailable}` to simplify a bit

**Comment by hannesj** [comment]
Also, should we show this only if the first vehicle has realtime information or any of them have it?

**Comment by csolem** [comment]
Would the start time from OTP be affected if one of the vehicles has realtime information, and is delayed?

**Pull request merged**